lib/code_climate/test_reporter/formatter.rb

```diff
@@ -36,6 +36,8 @@ def format(result)
         false
       end
 
+      private
```

Comment:
Style note -- this repo uses a rather unconventional, but consistent "outdented" `private`. So you'll want to push this back a tab.